### PR TITLE
Addition of ordered list style type upper-alpha

### PR DESCRIPTION
**Pull request by brenner-company**
### Overview
Concerning https://github.com/lblod/ember-rdfa-editor/issues/1194: the addition of ordered list style type `upper-alpha`.

### Challenges/uncertainties
I'm not sure of this change also has to be covered by the e2e test, but I've made a small addition of this (style) type within `e2e/files/ordered-list.html`. Though I can't seem to get the `list` tests running (getting a process related error from Playwright). If yes, can someone double check this for me?

### Checks PR readiness
- [x] npm lint
- [x] test (general)
- [x] e2e tests

### CODE REVIEW

File: addon/plugins/list/nodes/list-nodes.ts [hunk 1]
```diff
@@ -319,6 +323,8 @@ function renderEntry(entry: ListPathEntry) {
   const { style, pos } = entry;
   if (style === 'lower-alpha') {
     return indexToAlpha(pos);
+  } else if (style === 'upper-alpha') {
+    return indexToAlpha(pos).toUpperCase();
```
Comment (brenner-company) [inline addon/plugins/list/nodes/list-nodes.ts:327]: This seems the most simple way to accomplish this, but if extending `indexToAlpha` a bit to support upper case fits better here, just let me know.

Comment (elpoelma) [reply addon/plugins/list/nodes/list-nodes.ts:327]: this solution looks good to me!

**Review by elpoelma — COMMENTED** [review]
This solution looks good to me! I'm investigating the issues with playwright when using docker. Locally it runs without issues, but when running it in a docker container, I also get `process` errors.

**Comment by elpoelma** [comment]
@brenner-company I updated the playwright snapshot, so the e2e tests should run correctly on both CI and when running them locally. There are still issues with the playwright-docker setup, but that does not impact local/CI runs.

**Comment by brenner-company** [comment]
@elpoelma Thanks for the follow-up! I've added the missing changeset ✅

**Pull request merged**